tests/TypeInfer.builtins.test.cpp

```diff
@@ -566,6 +566,19 @@ TEST_CASE_FIXTURE(BuiltinsFixture, "string_format_tostring_specifier")
     LUAU_REQUIRE_NO_ERRORS(result);
 }
 
+TEST_CASE_FIXTURE(BuiltinsFixture, "string_format_tostring_specifier_type_constraint")
+{
+    CheckResult result = check(R"(
```

Comment:
Need to enable `LuauUnknownAndNeverType` too.

Reply:
Sure, though is there any reason tests don't *disable* all FFlags by default rather than use their default values? (Or run both)

Reply:
The tests run both with flags in their default state and with flags forced to true; this test must be passing regardless of the value of LuauUnknownAndNeverType.

Reply:
I still should enable LuauUnknownAndNeverType in here in case its default is changed to false, correct?

Reply:
Maybe I'm missing something, isn't the default for LuauUnknownAndNeverType already false?

Reply:
Would appear so now that I check. Weird!

Reply:
Oh, you know what. The reason why it passed in both states is just because no code requires the flag to be on in order to check if there is a UnknownTypeVar. It just simply does the latter, and they are only introduced into the environment by the former.